packages/hw-app-btc/src/Btc.js

```diff
@@ -288,7 +310,7 @@ export default class Btc {
     inputs: Array<{ trustedInput: boolean, value: Buffer }>,
     bip143?: boolean = false,
     overwinter?: boolean = false,
-    additionals: string[] = []
+    additionals?: Array<string>
```

Comment:
i didn't see this in previous review, but why not keeping like before?

`additionals: string[] = []` guarantee the array will be defaulted to `[]` when not provided. any reason to switch to `?: ` again?